### PR TITLE
backintime: update to 1.4.3.

### PR DESCRIPTION
**Pull request by thetredev**
#### Testing the changes
- I tested the changes in this PR: **YES**. The Qt GUI launches and I can setup backup targets. The CLI runs and doesn't complain, but I don't know how to use it so I assume it's fine also.

#### Local build testing
- I built this PR locally for my native architecture, `x86_64-glibc`

#### Questions to maintainers
- The package revision has been reset to `1`, because the minor version changed to `4`. Please tell me if this is correct.

#### Further Notes
- The patch `srcpkgs/backintime/patches/e1ae23ddc0b4229053e3e9c6c61adcb7f3d8e9b3.patch` has been deleted because it is included in backintime 1.4.1 (see https://github.com/bit-team/backintime/commit/e1ae23ddc0b4229053e3e9c6c61adcb7f3d8e9b3).

### CODE REVIEW

**Comment by Duncaen** [comment]
`backintime_package` defines a sub package by the main package name. The files are already in the package `backintime`, `backintime-qt` moves them out of it. If you want to keep the files in `backintime` they simply don't get moved, no need to define `backintime_package`. Not sure what files you actually want to move where though.

**Comment by thetredev** [comment]
Gotcha thanks. I'll try to remove `backintime_package` and see how it goes.

**Comment by thetredev** [comment]
That worked, thanks!

File: srcpkgs/backintime/template [hunk 1]
```diff
@@ -47,9 +46,5 @@ backintime-qt_package() {
 		vmove usr/share/bash-completion/completions/backintime-qt
 		vmove usr/share/dbus-1
 		vmove usr/share/doc/backintime-qt
-		vmove usr/share/doc/qt
-		vmove usr/share/icons
-		vmove "usr/share/man/man1/backintime-qt*"
```
Comment (Duncaen) [inline srcpkgs/backintime/template:52]: Putting the man page into the same package as the binary seems to make more sense.
The icons and the polkit file also seem to be more likely only used by the qt gui, what is the reason to change this?

Comment (thetredev) [reply srcpkgs/backintime/template:52]: `xbps-src` complained about `usr/share/doc/qt` not being there in the first place. Let me see where I got it wrong.

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Yeah, got the error again:
```
=> backintime-qt-1.4.1_1: running pkg_install ...
mv: cannot stat '/destdir//backintime-1.4.1/etc/dbus-1': No such file or directory
=> ERROR: backintime-qt-1.4.1_1: pkg_install: 'mv ${DESTDIR}/$files ${PKGDESTDIR}/${_targetdir}' exited with 1
=> ERROR:   in _vmove() at common/environment/setup/install.sh:228
=> ERROR:   in _noglob_helper() at common/environment/setup/install.sh:12
=> ERROR:   in pkg_install() at srcpkgs/backintime-qt/template:43
```
It tries to move the files out of `backintime` when it builds `backintime-qt`, so I'm confused now.

I reverted the whole `backintime-qt_package` function to the one from master.

Comment (thetredev) [reply srcpkgs/backintime/template:52]: The reason is that in fact `usr/share/doc` doesn't exist for `backintime-qt`. But as soon as I remove that line, I get:
```
=> backintime-qt-1.4.1_1: running pkg_install ...
mv: cannot stat '/destdir//backintime-1.4.1/etc/dbus-1': No such file or directory
=> ERROR: backintime-qt-1.4.1_1: pkg_install: 'mv ${DESTDIR}/$files ${PKGDESTDIR}/${_targetdir}' exited with 1
=> ERROR:   in _vmove() at common/environment/setup/install.sh:228
=> ERROR:   in _noglob_helper() at common/environment/setup/install.sh:12
=> ERROR:   in pkg_install() at srcpkgs/backintime-qt/template:4
```
where `etc/dbus-1` exists in `backintime-qt` but not in `backintime`.

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Oh, the `icons` and `polkit` actually do belong to `backintime`, not `backintime-qt`.

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Judging by the tree output above, I assume the PR changes are correct?

Comment (Duncaen) [reply srcpkgs/backintime/template:52]: The icons and the polkit file are both part of the qt part of it and are in the qt directory in the source. https://github.com/bit-team/backintime/tree/2f7b6099edf40366da1c1020385b300ac2793080/qt

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Well they are not in the `masterdir` anymore. Not sure how to go about that.

Comment (thetredev) [reply srcpkgs/backintime/template:52]: I think I got it now. Can you take another look please?

File: srcpkgs/backintime/template [hunk 2]
```diff
@@ -37,19 +37,11 @@ do_install() {
 
 backintime-qt_package() {
 	short_desc+=" - Qt GUI"
-	depends="backintime-${version}_${revision}
-	 python3-PyQt5 python3-PyQt5-dbus libnotify xdpyinfo"
+	depends="backintime-${version}_${revision} python3-PyQt5 python3-PyQt5-dbus"
 	pkg_install() {
-		vmove etc/dbus-1
-		vmove "usr/bin/backintime-qt*"
```
Comment (classabbyamp) [inline srcpkgs/backintime/template:44]: not moving the binaries seems to defeat the point of splitting it off

File: srcpkgs/backintime/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/backintime/template:52]: i think a lot of the removed `vmove`s need to be restored, see the diff: https://github.com/void-linux/void-packages/actions/runs/7305156960/job/19908370726?pr=47880#step:9:9

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Is that the diff between the new and previous XBPS package contents?

Comment (classabbyamp) [reply srcpkgs/backintime/template:52]: yes

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Cool, didn't know that we could do that. I'll get back to the PR after the holidays I guess.

**Comment by thetredev** [comment]
Updated to version v1.4.3

File: srcpkgs/backintime/template (hunk 1 above)
Comment (thetredev) [reply srcpkgs/backintime/template:52]: I still don't understand it... it won't compile until I remove all those `vmmove` lines. Without them, it compiles fine and the Qt GUI also works. The last commit works on my system. Maybe these files just aren't needed anymore by `backintime`? If there's another way, could you please provide a `diff` on `backintime-qt_package()` with your proposal? The one from `master` throws the errors I posted above for all lines that I removed in this PR.

Comment (classabbyamp) [reply srcpkgs/backintime/template:52]: the only ones that need to be removed are `vmove etc/dbus-1` and `vmove usr/share/doc/qt`, as they no longer exist in the package.

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Okay that worked. How did you figure that out if I may ask?

Comment (classabbyamp) [reply srcpkgs/backintime/template:52]: > How did you figure that out if I may ask?

I read the error message, checked what was in masterdir/destdir, removed the relevant line, `xbps-src clean`ed, retried

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Thanks! I'll keep that in mind for the next PRs.

Comment (thetredev) [reply srcpkgs/backintime/template:52]: Should be good now.

**Comment by thetredev** [comment]
Okay, now it's ready to be merged.

**Comment by thetredev** [comment]
bump. Rebased to lastest master

**Pull request merged**